webapp/src/route.tsx

```diff
@@ -27,7 +27,7 @@ type RouteProps = {
 
 function FBRoute(props: RouteProps) {
     const loggedIn = useAppSelector<boolean|null>(getLoggedIn)
-    const match = useRouteMatch<any>()
+    const match = useRouteMatch<any>({path: props.path})
```

Comment:
Because a <Route> hasn't been called at this point, we need to set the path when calling.

Reply:
This change may not be necessary based on this fix. 
https://github.com/mattermost/focalboard/pull/2598